lang/java/reef-bridge-client/src/main/avro/JobSubmissionParameters.avsc

```diff
@@ -49,6 +49,7 @@
         { "name": "jobSubmissionDirectoryPrefix", "type": "string" }
       ]
   },
+
```

Comment:
Nothing. Not sure why it is here. I think added a new line by mistake. Will change back.